### PR TITLE
Fix rendering of Info message

### PR DESCRIPTION
**Pull request by sridhargaddam**

### CODE REVIEW

**Comment by sridhargaddam** [comment]
Currently, notice is not getting rendered properly in the following pages
https://submariner-io.github.io/quickstart/openshift/
https://submariner-io.github.io/quickstart/openshiftgn/

This PR changes it to following 
Screenshot from 2020-05-05 17-55-39

File: src/content/quickstart/openshift/ready_clusters.md [hunk 1]
```diff
@@ -20,8 +20,4 @@ chmod a+x ./prep_for_subm.sh
 
 ```
 
-{{% notice info %}}
-
-Please note that the prep_for_subm.sh script has a few pre-requirements, you will need to install: **oc, aws-cli, terraform, and unzip**. 
-
-{{% /notice %}}
\ No newline at end of file
+> **_INFO_** Please note that prep_for_subm.sh script has few prerequisites. You will have to install: **oc, aws-cli, terraform, and unzip**.
```
Comment (tpantelis) [inline src/content/quickstart/openshift/ready_clusters.md:23]:
```suggestion
> **_INFO_** Please note that  **oc**, **aws-cli**, **terraform**, and **unzip** need to be installed before running the `prep_for_subm.sh` script.
```

**Comment by tpantelis** [comment]
The weird thing is that https://github.com/submariner-io/submariner-website/blob/master/src/content/quickstart/openshiftsd/_index.md has the same syntax yet it renders properly.

File: src/content/quickstart/openshift/ready_clusters.md (hunk 1 above)
Comment (sridhargaddam) [reply src/content/quickstart/openshift/ready_clusters.md]: Done. Thanks for reviewing @tpantelis

**Pull request merged**